controllers/pattern_controller.go

```diff
@@ -469,11 +450,42 @@ func (r *PatternReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		return err
 	}
 	r.driftWatcher, _ = newDriftWatcher(r.Client, mgr.GetLogger(), newGitClient())
+
+	if err = r.deployGitopsSubscription(customConfigFile); err != nil {
+		return err
+	}
```

Comment:
This isn't something we should be doing during setup

Reply:
ack